docker-compose.yml

```diff
@@ -1,11 +1,33 @@
 mysql:
-    extends:
-      file: common-compose.yml
-      service: mysql
-    links:
-      - consul:consul
+    image: autopilotpattern/mysql:latest
+    mem_limit: 4g
+    restart: always
+    # expose for linking, but each container gets a private IP for
+    # internal use as well
+    expose:
+      - 3306
+    labels:
+      - triton.cns.services=mysql
+    env_file: _env
+    environment:
+      - CONTAINERPILOT=file:///etc/containerpilot.json
+      - MANTA_PRIVATE_KEY
```

Comment:
We can (and probably should) eliminate this line because we're getting it from the `_env` file now.